cmake/modules/FindWine.cmake

```diff
@@ -33,12 +44,19 @@ FOREACH(FLAG ${WINEBUILD_FLAGS})
 		# WineHQ (/opt/wine-stable, /opt/wine-devel, /opt/wine-staging)
 		STRING(REPLACE "/lib64/wine/" "/lib/wine/" FLAG "${FLAG}")
 
```

Comment:
This logic doesn't work anymore because we drop `/wine/` before it and causes #4501.
Some possible solutions:

- Use a regular expression to match trailing `lib64`
- Remove `/wine/libwinecrt0.a.*` after replacing
- Always replace `lib64` with `lib` if it doesn't break other platforms

Reply:
Thanks for tracking this down. Fixed with 515fefa6ce734fcc5cf119b8eff8c3c0db65cdfc.